### PR TITLE
Implement Music Views

### PR DESCRIPTION
**Pull request by digawp**
To solve #14 
- [x] Music List View
- [x] Music Detail View
- [x] Music Manager

### CODE REVIEW

**Comment by digawp** [comment]
Only lacks thumbnail now but can merge for now

File: multimedia/views.py [hunk 1]
```diff
@@ -10,3 +11,18 @@ def book_list(request):
 def book_detail(request, isbn13):
     book = Book.objects.get(isbn13=isbn13)
     return render(request, 'multimedia/book_detail.html', {'book': book})
+
+def music_list(request):
+    musics = Music.objects.all()
+
+    for music in musics:
+        crews = Crew.objects.all(multimedia_id=music['id'])
```
Comment (9gix) [inline multimedia/views.py]: Overwrite `filter` instead of `all`

File: crew/managers.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
+from django.db import connection
+from django.db import models
+from django.core.urlresolvers import reverse
+
+from multimedia.utils import (
+    dictfetchall, dictfetchone
+)
+
+class CrewManager(models.Manager):
+    def all(self, *args, **kwargs):
```
Comment (9gix) [inline crew/managers.py]: Overwrite `filter` instead of `all`

**Comment by 9gix** [comment]
:+1:

**Pull request merged**